lib/toggles/feature/permissions.rb

```diff
@@ -24,7 +24,9 @@ def valid_for?(entities)
       rules.all? do |name, rule|
         entity = entities[name.to_sym]
 
-        if entity.class.ancestors.find { |ancestor| ancestor == Comparable }
+        if entity.nil?
+          return false
```

Comment:
This should really be its own if statement. It has nothing to do with the other conditional.